src/async-components/views/dialogs/security/CreateKeyBackupDialog.tsx

```diff
@@ -85,56 +65,27 @@ export default class CreateKeyBackupDialog extends React.PureComponent<IProps, I
     public async componentDidMount(): Promise<void> {
         const cli = MatrixClientPeg.get();
         const secureSecretStorage = await cli.doesServerSupportUnstableFeature("org.matrix.e2e_cross_signing");
-        this.setState({ secureSecretStorage });
 
-        // If we're using secret storage, skip ahead to the backing up step, as
-        // `accessSecretStorage` will handle passphrases as needed.
+        // If we're using secret storage, `accessSecretStorage` will handle passphrases as needed.
         if (secureSecretStorage) {
             this.setState({ phase: Phase.BackingUp });
             this.createBackup();
         }
     }
 
-    private onCopyClick = (): void => {
-        const successful = copyNode(this.recoveryKeyNode.current);
-        if (successful) {
-            this.setState({
-                copied: true,
-                phase: Phase.KeepItSafe,
-            });
-        }
-    };
-
-    private onDownloadClick = (): void => {
-        const blob = new Blob([this.keyBackupInfo.recovery_key], {
-            type: "text/plain;charset=us-ascii",
-        });
-        FileSaver.saveAs(blob, "security-key.txt");
-
-        this.setState({
-            downloaded: true,
-            phase: Phase.KeepItSafe,
-        });
-    };
-
     private createBackup = async (): Promise<void> => {
-        const { secureSecretStorage } = this.state;
         this.setState({
             phase: Phase.BackingUp,
             error: undefined,
         });
         let info;
```

Comment:
looks like maybe this needs a type annotation to avoid a typescript warning at line 100?